tools/CACertificates/ca-certs.ps1

```diff
@@ -708,18 +708,24 @@ function New-CACertsVerificationCert([Parameter(Mandatory=$TRUE)][string]$reques
     .PARAMETER isEdgeDevice
```

Comment:
The help text needs to be updated with the new `filePrefix` parameter that replaces `isEdgeDevice`.